src/features/XIT/CONTC/ConditionRow.vue

```diff
@@ -14,10 +14,11 @@ const eta = computed(() => {
   if (!isFinite(deadline)) {
     return '∞';
   }
-  if (deadline <= timestampEachSecond.value) {
+  const timestamp = timestampEachHour.value;
```

Comment:
This will straight up not work when the countdown goes down to seconds
<img width="404" alt="image" src="https://github.com/user-attachments/assets/80801c81-e365-4772-be2e-2a0f499f7a3d" />
The requirement is to show "Xm 59s" => "Xm 58s" => "Xm 57s" => "Xm 56s" => etc. If you use `timestampEachHour`, the countdown will freeze for an hour